go/ql/test/experimental/CWE-522-DecompressionBombs/DecompressionBombs.expected

```diff
@@ -0,0 +1,355 @@
+| test.go:268:3:268:6 | zstd | test.go:57:18:57:29 | selection of Body | test.go:268:3:268:6 | zstd | This decompression is $@. | test.go:57:18:57:29 | selection of Body | decompressing compressed data without managing output size |
+| test.go:270:3:270:6 | zstd | test.go:57:18:57:29 | selection of Body | test.go:270:3:270:6 | zstd | This decompression is $@. | test.go:57:18:57:29 | selection of Body | decompressing compressed data without managing output size |
+| test.go:277:3:277:6 | zstd | test.go:57:18:57:29 | selection of Body | test.go:277:3:277:6 | zstd | This decompression is $@. | test.go:57:18:57:29 | selection of Body | decompressing compressed data without managing output size |
+| test.go:283:3:283:8 | xzRead | test.go:57:18:57:29 | selection of Body | test.go:283:3:283:8 | xzRead | This decompression is $@. | test.go:57:18:57:29 | selection of Body | decompressing compressed data without managing output size |
+| test.go:287:2:287:8 | tarRead | test.go:57:18:57:29 | selection of Body | test.go:287:2:287:8 | tarRead | This decompression is $@. | test.go:57:18:57:29 | selection of Body | decompressing compressed data without managing output size |
+| test.go:287:2:287:8 | tarRead | test.go:57:18:57:29 | selection of Body | test.go:287:2:287:8 | tarRead | This decompression is $@. | test.go:57:18:57:29 | selection of Body | decompressing compressed data without managing output size |
+| test.go:287:2:287:8 | tarRead | test.go:57:18:57:29 | selection of Body | test.go:287:2:287:8 | tarRead | This decompression is $@. | test.go:57:18:57:29 | selection of Body | decompressing compressed data without managing output size |
+| test.go:287:2:287:8 | tarRead | test.go:57:18:57:29 | selection of Body | test.go:287:2:287:8 | tarRead | This decompression is $@. | test.go:57:18:57:29 | selection of Body | decompressing compressed data without managing output size |
+| test.go:287:2:287:8 | tarRead | test.go:57:18:57:29 | selection of Body | test.go:287:2:287:8 | tarRead | This decompression is $@. | test.go:57:18:57:29 | selection of Body | decompressing compressed data without managing output size |
+| test.go:287:2:287:8 | tarRead | test.go:57:18:57:29 | selection of Body | test.go:287:2:287:8 | tarRead | This decompression is $@. | test.go:57:18:57:29 | selection of Body | decompressing compressed data without managing output size |
+| test.go:287:2:287:8 | tarRead | test.go:57:18:57:29 | selection of Body | test.go:287:2:287:8 | tarRead | This decompression is $@. | test.go:57:18:57:29 | selection of Body | decompressing compressed data without managing output size |
+| test.go:287:2:287:8 | tarRead | test.go:57:18:57:29 | selection of Body | test.go:287:2:287:8 | tarRead | This decompression is $@. | test.go:57:18:57:29 | selection of Body | decompressing compressed data without managing output size |
```

Comment:
There are 8 results for this sink. But there are 16 places where `tarRead` is assigned to. So shouldn't there be 16? Unfortunately it's hard to see what's going on because they all show the same source and sink. One way to make it easier would be to use a different source for each conditional clause in `TarDecompressor`, e.g. by making `reqest` a parameter to that function and using `request.Body` in each conditional clause.